airbyte-server/src/main/java/io/airbyte/server/handlers/InternalWorkerHandler.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.server.handlers;
+
+import io.airbyte.api.model.generated.InternalOperationResult;
+import io.airbyte.api.model.generated.SetTemporalWorkflowInAttemptRequestBody;
+import io.airbyte.scheduler.persistence.JobPersistence;
+import java.io.IOException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class InternalWorkerHandler {
```

Comment:
nit: would a better name be `internal` handler?

We should add a javadocs here to make clear this is for route internal to Airbyte operation.

Reply:
@xiaohansong fyi